frank-flow/src/frontend/src/app/shared/components/file-tree/file-tree.component.ts

```diff
@@ -172,6 +172,7 @@ export class FileTreeComponent implements OnInit, OnDestroy {
       const item: File = JSON.parse(itemValue);
 
       if (this.filesAreEqual(this.currentFile, item)) {
+        this.currentFileService.resetCurrentDirectory()
```

Comment:
In the next code block the current file gets set anyway. For what else purpose is this needed? I guess it's to reset the current Directory when a file is switched to another one. But shouldn't it then be placed at the code where that happens? I.e. currentFileService.switchToFileTreeItem?

Reply:
I added this when I was trying to get the delete to work. Ill test if its still needed